core/src/main/java/org/elasticsearch/search/aggregations/bucket/terms/AbstractTermsParametersParser.java

```diff
@@ -74,8 +75,9 @@ public void parse(String aggregationName, XContentParser parser, SearchContext c
                 currentFieldName = parser.currentName();
             } else if (vsParser.token(currentFieldName, token, parser)) {
                 continue;
-            } else if (incExcParser.token(currentFieldName, token, parser)) {
-                continue;
+                // } else if (incExcParser.token(currentFieldName, token,
+                // parser)) {
+                // continue;
```

Comment:
This class is actually not used anymore and should have been deleted. Will delete now